bundle/bundle.go

```diff
@@ -43,6 +45,33 @@ type Bundle struct {
 	AutoApprove bool
 }
 
+const ExtraIncludePathsKey string = "DATABRICKS_BUNDLE_INCLUDE_PATHS"
+
+// Get extra include paths from environment variable
+func GetExtraIncludePaths() []string {
+	value, exists := os.LookupEnv(ExtraIncludePathsKey)
+	if !exists {
+		return []string{}
+	}
+	var sep = ":"
+	if runtime.GOOS == "windows" {
+		sep = ";"
+	}
+	return strings.Split(value, sep)
+}
+
+// Try loading bundle config from one of the extra include paths from the environment variable
+// If not able to load, do not return an error, just return nil
+func tryLoadFromExtraIncludePaths() *Bundle {
```

Comment:
Based on the logic env variable is a bit confusing. My assumption was that this env variable is used to specify paths where othe bundle configs can be found but instead the logic seems to be closer to DATABRICKS_BUNDLE_ROOTS which defines a list of potential bundle roots and first one going to be used

Reply:
This function is supposed to be called ONLY when BUNDLE_ROOT is already known. I have added a explicit check for this.

Reply:
@kartikgupta-db but why do we need to do this even? the point of bundle root is to define where the `bundle.yml` is, why do we want to support loading bundle configs from different locations?
If we do so, it also means that `bundle.yml` won't be version controlled together with the bundle code itself.

@pietern any thoughts on this?